src/index.js

```diff
@@ -142,11 +142,12 @@ listTasks.addEventListener('change', (e) => {
 const clearBtn = document.querySelector('#clearBtn');
 
 clearBtn.addEventListener('click', () => {
-  const UncompeledTasks = list.filter((task) => task.completed === false);
-  UncompeledTasks.forEach((task, i) => { (task.index = i + 1); });
-  list = UncompeledTasks;
-  displayTasks(list);
-  UpdateStorage(list);
+  // const UncompeledTasks = list.filter((task) => task.completed === false);
+  // UncompeledTasks.forEach((task, i) => { (task.index = i + 1); });
+  // list = UncompeledTasks;
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.